Pod/Classes/MenuView.swift

```diff
@@ -85,21 +85,21 @@ public class MenuView: UIScrollView {
     
     // MARK: - Lifecycle
     
-    internal init(menuItemTitles: [String], menuItemDescs: [String], options: PagingMenuOptions) {
+    public init(menuItemTitles: [String], menuItemDescs: [String], options: PagingMenuOptions) {
```

Comment:
As long as it's a static header, I think its fine. However I'm not sure everyone use it in a same way. So could you revert this change if its not relevant to multiline text mode?
I'm taking care of this feature.